Utilities/build-script-helper.py

```diff
@@ -112,6 +116,8 @@ def handle_invocation(swift_exec, args):
   print('Cleaning ' + args.build_path)
   shutil.rmtree(args.build_path, ignore_errors=True)
 
+  env['SWIFT_EXEC'] = '%sc' % (swift_exec)
```

Comment:
I had to add this for [the same reason as `swift-driver`](https://github.com/apple/swift-driver/pull/743#discussion_r663638507), so see that note.